lib/pbench/test/unit/common/test_config.py

```diff
@@ -81,34 +81,41 @@ def test_custom_logger(self, monkeypatch):
         config = PbenchConfig("pbench.cfg")
         assert config.logger_type == logger_type
         assert config.log_dir == log_dir
+        assert config.log_using_caller_directory is False
         assert config.default_logging_level == logging_level
         assert config.log_fmt == log_format
+        assert config.TZ == "UTC"
 
     def test_bad_logger_hostport(self, monkeypatch):
         """Show that PbenchConfig raises 'BadConfig' and 'ValueError' when
         the configuration contains no option or an empty string respectively"""
 
         logger_type = "hostport"
+        expected_error_msg = "No option 'logger_host' in section: 'logging'"
         self.logger_test_setup(monkeypatch, {"logging": {"logger_type": logger_type}})
         with pytest.raises(BadConfig) as exc:
             PbenchConfig("pbench.cfg")
-        expected_error_msg = "No option 'logger_host' in section: 'logging'"
-        assert expected_error_msg in str(exc)
+        assert expected_error_msg in str(exc.value)
+
+        expected_error_msg = "ValueError('logger_host', 'logging')"
         self.logger_test_setup(
             monkeypatch, {"logging": {"logger_type": logger_type, "logger_host": ""}}
         )
         with pytest.raises(ValueError) as exc:
             PbenchConfig("pbench.cfg")
-        expected_error_msg = "ValueError('logger_host', 'logging')"
+        assert exc.value.args[0] == "logger_host"
+        assert exc.value.args[1] == "logging"
+
         logger_host = "logger.example.com"
+        expected_error_msg = "No option 'logger_port' in section: 'logging'"
```

Comment:
We're not actually using the value assigned at line 100.  So, we should either omit that line or rework lines 106-107 to be more like line 98 (or 117).  (That is, unless there is some reason to do it differently here, I would make the code here match what it does in those other places.)

Ditto for line 129 (and 132-133).